packages/angular/cli/tasks/install-package.ts

```diff
@@ -155,6 +155,8 @@ export function runTempPackageBin(
     env: {
       ...process.env,
       NG_DISABLE_VERSION_CHECK: 'true',
+      NG_FORCE_TTY: 'false',
```

Comment:
Thinking about it some more, we may want to add interactivity to the update process so just disabling analytics may be the better long-term option.

Reply:
Updated.